exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Thing.java

```diff
@@ -1,8 +1,40 @@
 package com.scala.exp.android.sdk.model;
 
+import java.util.List;
+
 /**
  * Created by Cesar Oyarzun on 10/28/15.
  */
 public class Thing extends AbstractModel {
 
+    private List<Zone> zones;
+    private Location location;
+    private Experience experience;
+
+    public void setZones(List<Zone> zones) {
+        this.zones = zones;
+    }
+
+    public List<Zone> getZones() {
+        return zones;
+    }
+
+    public Location getLocation() {
+        return location;
+    }
+
+    public void setLocation(Location location) {
+        this.location = location;
+    }
+
+
+    public Experience getExperience() {
```

Comment:
I missed this on the iOS SDK, it needs to be removed from there as well: https://github.com/ScalaInc/exp-ios-sdk/blob/develop/Pod/Classes/Thing.swift#L55